web/packages/teleport/src/AppLauncher/AppLauncher.tsx

```diff
@@ -75,15 +76,25 @@ export function AppLauncher() {
         }
       }
 
+      let requiredApps = resolvedApp.requiredApps || [];
+      if (isRedirectFlow !== null) {
```

Comment:
Specifically checking for null because `redirectFlow` is got from `queryParams.get` which returns null or string. (also, the double negation throws eslint here based on `string` type so, i'll leave this one)